src/coreclr/jit/importer_vectorization.cpp

```diff
@@ -88,17 +88,23 @@ static GenTreeHWIntrinsic* CreateConstVector(Compiler* comp, var_types simdType,
 #ifdef TARGET_XARCH
     if (simdType == TYP_SIMD32)
     {
-        GenTree* long1 = comp->gtNewIconNode(*(ssize_t*)(cns + 0), TYP_LONG);
-        GenTree* long2 = comp->gtNewIconNode(*(ssize_t*)(cns + 4), TYP_LONG);
-        GenTree* long3 = comp->gtNewIconNode(*(ssize_t*)(cns + 8), TYP_LONG);
-        GenTree* long4 = comp->gtNewIconNode(*(ssize_t*)(cns + 12), TYP_LONG);
+        ssize_t fourLongs[4];
+        CopyMemory(fourLongs, cns, sizeof(ssize_t) * 4);
```

Comment:
The only use of this function in the JIT is in importer_vectorization.cpp. Why not use `memcpy`?

Reply:
I guess it's just an alias for memcpy anyway. I probably just noticed it in some VM's code but since we don't use it in jit anywhere I converted it back to memcpy